src/systems/emoji-system.js

```diff
@@ -0,0 +1,55 @@
+import { paths } from "./userinput/paths";
+import { emojis, spawnEmojiInFrontOfUser } from "../components/emoji";
+
+const actions = [
+  paths.actions.spawnEmoji0,
+  paths.actions.spawnEmoji1,
+  paths.actions.spawnEmoji2,
+  paths.actions.spawnEmoji3,
+  paths.actions.spawnEmoji4,
+  paths.actions.spawnEmoji5,
+  paths.actions.spawnEmoji6
+];
+
+const EMOJI_RATE_LIMIT = 1000;
```

Comment:
I'd very much appreciate it if we removed or reduced this limit. Rapid emoting is definitely something I want to be able to do. Pre-empting a couple concerns:
- If performance is an issue here, it seems easy to address with our batching system.
- If abuse is an issue here, this seems like the wrong cut-point for addressing it.